src/api/attack/util.rs

```diff
@@ -793,88 +653,3 @@ pub fn get_attacker_types(conn: &mut PgConnection) -> Result<HashMap<i32, Attack
         })
         .collect::<HashMap<i32, AttackerType>>())
 }
-
-pub fn get_buildings(
```

Comment:
Why is this removed?

Reply:
This was unused after removing diffuser and drone